lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -3932,20 +3932,20 @@ class SPIRVTaskSequenceCreateINTELInst : public SPIRVTaskSequenceINTELInst {
         ClusterMode >= -1 && ClusterMode <= 1, SPIRVEC_InvalidInstruction,
         InstName + "\nClusterMode valid values are -1, 0, 1.\n");
 
-    const uint32_t GetCapacity =
+    const int GetCapacity =
         static_cast<SPIRVConstant *>(
             const_cast<SPIRVTaskSequenceCreateINTELInst *>(this)->getOperand(3))
             ->getZExtIntValue();
     SPVErrLog.checkError(
-        GetCapacity, SPIRVEC_InvalidInstruction,
+        GetCapacity >= 0, SPIRVEC_InvalidInstruction,
         InstName + "\nGetCapacity must be unsigned 32 bit integer.\n");
```

Comment:
"GetCapacity must be an unsigned 32-bit integer" is better.

Reply:
Applied